apis/apps/v1alpha1/opsrequest_types.go

```diff
@@ -185,12 +189,23 @@ type SpecificOpsRequest struct {
 	// +optional
 	Backup *Backup `json:"backup,omitempty"`
 
+	// Specifies the parameters to backup a Cluster.
```

Comment:
Add "Deprecated: since v0.9, use `backup` instead" in the comment.
Similarly, include deprecation notes for other fields.